ui/src/timeMachine/actions/index.ts

```diff
@@ -704,3 +665,26 @@ export const loadNewVEO = (dashboardID: string) => (
   }
   // no need to refresh variable values since there is no query in a new view
 }
+
+export const loadCustomQueryState = () => (
```

Comment:
is this specifically for checks? it might be good to name this function in a way that makes it clear that it's for checks specifically, if that's the case of course.

Reply:
thanks!